openedx_webhooks/info.py

```diff
@@ -152,6 +153,9 @@ def is_committer_pull_request(pull_request: PrDict) -> bool:
 
     repo = pull_request["base"]["repo"]["full_name"]
     org = repo.partition("/")[0]
+    # Check if this is a named release branch
+    # compare_url is "https://api.github.com/repos/octocat/Hello-World/compare/{base}...{head}"
+    open_release = "open-release" in pull_request["base"]["compare_url"].split("...")[0]
```

Comment:
Instead of a boolean "open-release" key, why not a "branch" key with a pattern for a value?

Reply:
I just edited the description to say that was a deliberate choice because I think it's easier and more readable, but I'm open to disagreements.

Reply:
(particularly, this matches the bot's behavior which is hardcoded)

Reply:
It feels to me like it's not much more to use a pattern here, and will perhaps be more understandable in people.yaml, since "branch: open-release/*" is pretty clear in Git terms, but "open-release: true" is less so.